spec/models/consultation_spec.rb

```diff
@@ -138,4 +138,37 @@
       expect(consultation.neighbour_responses_by_summary_tag).to eq({ "objection" => 1, "supportive" => 2, "neutral" => 1 })
     end
   end
+
+  describe "publicity_active?" do
```

Comment:
Would a test for the behaviour when the end date is today be useful?

Reply:
I think we'd still want this warning if the end date is today as technically it should go on until the end of day right

Reply:
Yeah, I think the behaviour seems correct: "greater than today".

Reply:
Added a spec to compare the actual day rather than time in this instance.